libfwupdplugin/fu-udev-device.c

```diff
@@ -1465,10 +1465,11 @@ fu_udev_device_set_physical_id(FuUdevDevice *self, const gchar *subsystems, GErr
 		}
 		physical_id = g_strdup_printf("PCI_SLOT_NAME=%s", tmp);
 	} else if (g_strcmp0(subsystem, "usb") == 0 || g_strcmp0(subsystem, "mmc") == 0 ||
-		   g_strcmp0(subsystem, "i2c") == 0 || g_strcmp0(subsystem, "platform") == 0 ||
+		   g_strcmp0(subsystem, "i2c") == 0 || g_strcmp0(subsystem, "i2c-dev") == 0 ||
```

Comment:
Nit: Can you please split this into two commits where one of them has each one alphabetically on its own line and the other adds a new match?

Reply:
it looks like `clang-format` enforced the lines combined unless we do `clang-format off`. Maybe I can add `i2c-dev` at the tail as a minimum change.

Reply:
I guess it's fine they combined, but I think sticking to alphabetical order is ideal.

Reply:
ok, added another commit to sort the subsystems.

Reply:
i2c-dev isn't really a subsystem tho

Reply:
ACK, removed the addition.